config/default.yml

```diff
@@ -783,6 +783,11 @@ PrePush:
     description: 'Running RSpec test suite'
     required_executable: 'rspec'
 
+  Minitest:
+    enabled: false
+    description: 'Running Minitest test suite'
+    command: ['bundle', 'exec', 'rake', 'test']
```

Comment:
Maybe submit a PR, so we can discuss there?

On Wed, Feb 10, 2016 at 2:59 PM, David Rodríguez <notifications@github.com>
wrote:

> In config/default.yml
> <https://github.com/brigade/overcommit/pull/304#discussion_r52532847>:
>
> > @@ -783,6 +783,11 @@ PrePush:
> >      description: 'Running RSpec test suite'
> >      required_executable: 'rspec'
> >
> > +  Minitest:
> > +    enabled: false
> > +    description: 'Running Minitest test suite'
> > +    command: ['bundle', 'exec', 'rake', 'test']
>
> Actually the previous version runs tests twice. This should work instead
>
> bundle exec ruby -Ilib:test -rminitest/autorun -e 'Dir["test/**/*_test.rb"].each { |test| require File.expand_path(test) }'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brigade/overcommit/pull/304/files#r52532847>.
>